chipy_org/apps/job_board/templates/job_post_list.html

```diff
@@ -0,0 +1,35 @@
+{% extends "site_base.html" %}
+
+
+{% block body %}
+
+<h2>Job Posts</h2>
+    <ul>
+        
+        {% for post in job_posts %}
+
+            {% ifchanged post.is_sponsor %}
+                {% if post.is_sponsor %}
+                    <h3>Sponsored Job Posts</h3>
+                {% else %}
+                    <h3>Regular Job Posts</h3>
+                {% endif %}
+            {% endifchanged %}
+
+            <li><a href="{% url 'job-post-detail' pk=post.pk %}">{{post.position}} at {{ post.company_name }}</a>, 
+                    
+                Posted  
+                {% if post.days_elapsed == 0 %} 
+                    today 
+                {% elif post.days_elapsed == 1 %}
+                    1 day ago 
+                {% else %}
+                    {{post.days_elapsed}} days ago
```

Comment:
For the second two conditions you can switch to `pluralize`.

```suggestion
                    {{post.days_elapsed}} day{{ post.days_elapsed|pluralize }} ago
```

https://docs.djangoproject.com/en/3.0/ref/templates/builtins/#pluralize